app/components/util/ranking.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+
+export default function Ranking(rankingProps) {
+  const responseOptions = ['Most', 'Neutral', 'Least']
+  const displayPropOptions = ['block', 'small', 'medium', 'large']
+
+  //Define new and inherited classes
+  let classes = ['radial_ranking', ...(className ? className : [])]
+
+  //Isolate props and set initial state
+  const { disabled, defaultValue, className, onSelected, ...otherProps } = rankingProps
+  let [response, setResponse] = useState(responseOptions.includes(defaultValue) ? defaultValue : '')
+
+  displayPropOptions.forEach(prop => {
```

Comment:
I've seen block and large, etc in projects but we are not using that as a pattern. Unless you have a need in mind, lets simplify it out.